ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -282,12 +283,25 @@ let test_vdi_after_data_destroy () =
 (* check VDI.data_destroy throws VDI_NOT_MANAGED if managed:false *)
 let test_vdi_managed_data_destroy () =
   let __context = Test_common.make_test_database () in
-  let _,vDI = make_mock_server_infrastructure ~__context in
+  let sR,vDI = make_mock_server_infrastructure ~__context in
+  register_smapiv2_server ~vdi_data_destroy:(fun _ ~dbg ~sr ~vdi -> ()) (Db.SR.get_uuid ~__context ~self:sR);
   Db.VDI.set_managed ~__context ~self:vDI ~value:false;
   OUnit.assert_raises ~msg:"VDI.data_destroy only works on managed VDI"
     Api_errors.(Server_error (vdi_not_managed, [Ref.string_of vDI]))
     (fun () -> Xapi_vdi.data_destroy ~__context ~self:vDI)
 
+(* verify message forwarding for VDI.list_changed_blocks *)
+let test_vdi_list_changed_blocks () =
+  let __context = Test_common.make_test_database () in
+  let sR,vDI = make_mock_server_infrastructure ~__context in
+  let vdi_to = Test_common.make_vdi ~__context ~sR () in
+  let list_changed_blocks_string = "listchangedblocks000" in
+  register_smapiv2_server ~vdi_list_changed_blocks:(fun _ ~dbg ~sr ~vdi_from ~vdi_to -> list_changed_blocks_string) (Db.SR.get_uuid ~__context ~self:sR);
```

Comment:
One additional thing that we could verify here is that we pass the correct parameters to the SMAPIv2 function. Here is an example of this test: https://github.com/xapi-project/xen-api/blob/4790bdb8709b25099fd09191160ada6ca7f4ad9e/ocaml/xapi/test_vdi_cbt.ml#L60